src/sentry/issues/endpoints/group_similar_issues_embeddings.py

```diff
@@ -82,9 +83,11 @@ def get(self, request: Request, group) -> Response:
         stacktrace_string = ""
         if latest_event and event_content_has_stacktrace(latest_event):
             grouping_info = get_grouping_info(None, project=group.project, event=latest_event)
-            stacktrace_string = get_stacktrace_string(grouping_info)
+            stacktrace_string = get_stacktrace_string_handle_system_frame_exception(
```

Comment:
do we need to call this function with the extra metric logging? or can we just call `get_stacktrace_string` directly?

Reply:
Yeah, that's a good callout. I was using the function as an "exception catcher", but it makes more sense to use it to add metrics and catch the exception without calling the function